src/autorestSession.ts

```diff
@@ -34,6 +34,7 @@ export interface AutorestOptions {
   generateTest?: boolean;
   batch?: [string, any][];
   generateSample?: boolean;
+  monoRepo?: boolean;
```

Comment:
If we are going this route, I'd prefer a more explicit flag, like `azure-sdk-for-js` which is the only place where the generated code would work. Otherwise it may be mistaken that it is a flag that should be used for **any** mono-repo

Reply:
The `mono-repo` comes from @sarangan12 's comment in https://github.com/Azure/autorest.typescript/issues/1259 . Cloud you please also talk with @sarangan12 about it? thanks

Reply:
@dw511214992 I agree with Jose. Though we refer azure-sdk-for-js repo as mono repo, using it in the flag might confuse people if they are outside our org.

Reply:
okay, I will change it to `azure-sdk-for-js`